packages/whats-new/src/style.scss

```diff
@@ -6,155 +6,175 @@
 		display: flex;
-		justify-content: center;
+		justify-content: space-between;
 		align-items: center;
 		background: var( --studio-white );
 		border-top: none;
 		box-sizing: border-box;
-		min-height: $wpcom-modal-footer-height;
 
 		@media ( min-width: $wpcom-modal-breakpoint ) {
-			padding: $wpcom-modal-footer-padding-v 0;
 			position: absolute;
-			bottom: $wpcom-modal-footer-height * -1;
-			height: $wpcom-modal-footer-height;
+			width: $wpcom-modal-inner-content-width;
+			height: auto;
+			bottom: 0;
 			left: 0;
+			padding: $wpcom-modal-content-spacing;
 		}
 
-		.components-button {
-			background-color: var( --color-surface );
-			color: var( --color-neutral-70 );
-			border-color: var( --color-neutral-10 );
-			border-radius: 2px;
-			border-style: solid;
-			border-width: 1px;
-			box-sizing: border-box;
-			font-weight: 400;
-			line-height: 22px;
-			padding: 12px 16px;
-			text-decoration: none;
-			vertical-align: top;
-
-			&:active,
-			&.is-active {
-				border-width: 1px;
-			}
+		@media ( max-width: $wpcom-modal-breakpoint ) {
+			padding: 16px;
+		}
 
-			&:hover {
-				border-color: var( --color-neutral-20 );
+		.guide__buttons {
+			.components-button {
+				background-color: var( --color-surface );
 				color: var( --color-neutral-70 );
+				border-color: var( --color-neutral-10 );
+				border-radius: 2px;
+				border-style: solid;
+				border-width: 1px;
+				box-sizing: border-box;
+				font-weight: 400;
+				line-height: 22px;
+				padding: 6px 16px;
+				text-decoration: none;
+				vertical-align: top;
+
+				&:active,
+				&.is-active {
+					border-width: 1px;
+				}
+
+				&:hover {
+					border-color: var( --color-neutral-20 );
+					color: var( --color-neutral-70 );
+				}
+
+				.accessible-focus &:focus {
+					border-color: var( --color-primary );
+					box-shadow: 0 0 0 2px var( --color-primary-light );
+				}
 			}
 
-			.accessible-focus &:focus {
-				border-color: var( --color-primary );
-				box-shadow: 0 0 0 2px var( --color-primary-light );
-			}
-		}
-
-		.components-guide__forward-button {
-			background-color: var( --color-accent );
-			border-color: var( --color-accent );
-
-			color: var( --color-text-inverted );
-			font-size: $font-body-small;
+			.guide__forward-button {
+				background-color: var( --color-accent );
+				border-color: var( --color-accent );
 
-			&:hover,
-			&:focus {
-				background-color: var( --color-accent-60 );
-				border-color: var( --color-accent-60 );
 				color: var( --color-text-inverted );
+				font-size: $font-body-small;
+
+				&:hover,
+				&:focus {
+					background-color: var( --color-accent-60 );
+					border-color: var( --color-accent-60 );
+					color: var( --color-text-inverted );
+				}
+
+				.accessible-focus &:focus {
+					box-shadow: 0 0 0 2px var( --color-accent-light );
+				}
 			}
 
-			.accessible-focus &:focus {
-				box-shadow: 0 0 0 2px var( --color-accent-light );
+			.guide__back-button {
+				margin-right: 20px;
```

Comment:
In keeping with the other dual button combos on the page and in the home it may be better to remove the border around the back button. You can see this in the Welcome Guide. Maybe @nuriapenya can weigh in on this though?

Reply:
@arcangelini After revising all the designs from Help Center & Next Steps (https://createp2.wordpress.com/2022/05/17/help-guidance-v1-i2-introducing-next-steps/) I believe we will be using this dual button combo more often in guided tours, steps, etc. So I guess the main idea here is to make this secondary button more obvious and aligned with the layout - so, in this sense, the implementation of Kosta is correct 👌🏼